src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -114,7 +114,7 @@ public function denormalize($data, $class, $format = null, array $context = [])
         try {
             return \DateTime::class === $class ? new \DateTime($data, $timezone) : new \DateTimeImmutable($data, $timezone);
         } catch (\Exception $e) {
-            throw new NotNormalizableValueException($e->getMessage(), $e->getCode(), $e);
+            // let the validator show the error and do not throw 500.
```

Comment:
I would remove `500` as its really http context specific.

Just

`let the validator show the error and do not throw here`?